README.md

```diff
@@ -1,5 +1,84 @@
+
+Once you have successfully logged into Edlab, upload a file named ______.txt where the blank space is replaced with your SPIRE username using the ‘put’ command. Then, use the ‘ls’ command to view the file directory, and attach a screenshot to your solution.
+
+## Part 2: Pointers (20 Points)
+When programming in C++, we will frequently make use of pointers. Every variable that we make use of in this class has a memory location, which we can access through the ‘&’ operator. When we have such a memory location, we can then access its stored value through the ‘*’ operator. We use this same operator when declaring our pointers in the code. As an example, take the following code:
+
+```
+int main() {
+
+	int original = 1; //variable that stores an integer value of 1
+	int *pointer = &one; //variable that stores the memory address of ‘original’
+
+	cout << original; //prints out the stored value of original: “1”
+	cout << pointer; //prints out the stored value of pointer (some memory address)
+	cout << *pointer; //prints out the stored value of the variable at the location of pointer (original): “1”
+
+	//Question code goes here
+
+	return 1;
+}
+```
+
+As can be seen in the code, pointers allow for a lot of memory reference to be carried out in C++. We can also combine multiple operators (though it is usually very inefficient to do so). This means that ‘&(*pointer)’ will result in just ‘pointer’, but there could be problems if ‘*(*pointer)’ is called, since that would be looking at the data stored with memory address of ‘*pointer’.
+
+Answer the following questions about pointers, with code referencing the main method written above at the designated line:
+1. (3 points) Is ‘*pointer’ equal to ‘&one’?
+1. (3 points) Would the code ‘&(*(*(&(&(*(&(*one)))))))’ be able to run?
+1. (3 points) Is ‘&pointer’ equal to ‘&(&one)’?
+1. (3 points) If the code ‘original = 3’ runs, would the value of ‘*pointer’ change as well? Why or why not?
+1. (8 points) Write code that would create a variable named ‘super_pointer’ that stores the memory address for the ‘pointer’ variable.
+
+## Part 3: Maps (20 Points)
+In C++, maps are a unique data type that allows for the mapping of one set of keys to a different set of values. An example of using a map is shown below, mapping a few numbers to their respective Pokémon.
+
+```
+int main(){
+
+	map<int, string> pokedex;
+
+	pokedex.insert(pair<int, string>(2, “Ivysaur”));
+	pokedex.insert(pair<int, string>(5, “Charizard”));
+	pokedex.insert(pair<int, string>(6, “Squirtle”));
+	pokedex.insert(pair<int, string>(25, “Pikachu”));
+	pokedex.insert(pair<int, string>(39, “Jigglypuff”));
+
+	for (pokemon = pokedex.begin(); pokemon != pokedex.end(); pokemon ++) {
+		cout << pokemon->first << ‘: ‘ << pokemon->second << ‘\n’
+	}
+
+	//Question code goes here
+
+	return 151;
+}
+
+```
```

Comment:
I might be wrong, but isn't `map[key]=value` more commonly used than `map.insert(pair(key, value))`? Sure they are not equivalent, but I think the former way should also be mentioned here. What do you think?